### ISSUE TITLE
Things left before we can merge

### ISSUE THREAD

**Issue by btopro** [elmsln/elmsln#391]
MUST HAVES
- [x] remixables button needs added / some replacement
- [x] accessibility options needs added back in
- [x] log in needs fixed if viewing as anonymous (currently profile tab shows nothing)
- [x] Outline Designer AJAX calls currently throw an error (even tho they worked), Must be a notice on the backend permeating through.
- [x] edit button shows up at times when its not possible (visually, not a security issue)
- [x] Studio has hard break on assignment listing
- [x] Comply has hard break on entityform references
- [x] edit icon needs a label
- [x] as part of #379 it shouldn't say home but should say name of tool (Studio, etc)
- [x] template file for the modal block of remixable needs fixed
- [x] context needs updated for foundation_access_ux so remixable block positioned through there

Add anything else you find that we have to wait on before merging. @heyMP

**Comment by btopro** [comment]
OD was a false positive so resolved

**Comment by btopro** [comment]
can't replicate prev studio issue either

**Comment by btopro** [comment]
@michael-collins any thoughts on where print, accessibility, PDF, speed reader, contast, etc should go.

Also where the remixables should go given the new interface styling currently doesn't have a place for them (because we've cleaned up so much).

**Comment by michael-collins** [comment]
@btopro embed/remix icon should be to the right of settings icon. I uploaded it to slack. PDF eliminates need for a print button. I would have a download pdf area at the bottom of the page (along with video transcripts should those exist). Accessibility stuff like contrast should probably go in account settings. We don't need to see that all the time, and it would probably need to be saved information anyway. If you don't like that idea, then put my icon back on the top right. We'll have 4 icons on the page if you are a content editor which is a lot, but not the end of the world. Otherwise, we can have an edit button, and then an ellipsis icon that has settings, embed/remix, and accessibility.

**Comment by btopro** [comment]
Was thinking of print as more so an alternate form of the page. There's speedreader, a voice reader, transcripts, PDF, print and someday maybe even epub formats. Dono if these go together or in two separate areas.

**Comment by btopro** [comment]
Contrast / text size I agree tho most sites have these immediately visible

**Comment by btopro** [comment]
icons added back to the right of the gear for now

**Comment by btopro** [comment]
Had to do manually because of some merge conflicts but this branch has been merged into master! Big UX overhaul, still clean up to do but the heavy lifting is in place and upgrade paths tested in vagrant worked. next step is to test in a pull down as well as clean environment build

**Comment by michael-collins** [comment]
Can I have a screen shot of what this looks like?

**Comment by btopro** [comment]
![2015-12-15_21-31-04](https://cloud.githubusercontent.com/assets/329735/11830439/309eaeca-a373-11e5-8e57-bac2e13d37aa.png)

**Comment by michael-collins** [comment]
@btopro Why have you not used the embed icon I made yesterday? That other icon is meaningless given it's a bunch of arrows pointing to people in a circle. Also, why was the word "edit" added to the pencil icon? I don't remember discussing that change.

Here are my recommendations. Remove the edit label. Add the Remix icon I designed to convey actions. Under that icon, put options for PDF, Print, and other functions you mentioned via a dropdown like settings.

In this way, it essentially becomes the icon/menu that allows you to engage with the page content. Print, remix, whatever.

**Comment by btopro** [comment]
this issue is closed, start a new one. I also have no uable formats for the ridiculously confusing share / embed / print to pdf / print / ePub export button that somehow conveys all of that.

**Comment by btopro** [comment]
http://iconmonstr.com/?s=share prototypes look nothing like any of these common ones

**Comment by rickhumphries** [comment]
@btopro @michael-collins In my mind, there should be a separate export icon, under which export as pdf, print, epub would all sit. You could also add 'export to github' or 'export as XML' as options for staff/admin only under here too perhaps?
